samples/40.timex-resolution/src/main/java/com/microsoft/bot/sample/timex/resolution/Ambiguity.java

```diff
@@ -80,9 +88,12 @@ public static void timeAmbiguity() {
         }
     }
 
+    /**
+     * Run the recognizer.
+     */
     public static void dateTimeAmbiguity() {
-        // Run the recognizer.
-        List<ModelResult> results = DateTimeRecognizer.recognizeDateTime("It will be ready Wednesday at 5 o'clock.", Culture.English);
+        List<ModelResult> results = DateTimeRecognizer.recognizeDateTime(
+            "It will be ready Wednesday at 5 o'clock.", Culture.English);
```

Comment:
```suggestion
        List<ModelResult> results = DateTimeRecognizer.recognizeDateTime(
            "It will be ready Wednesday at 5 o'clock.",
            Culture.English);
```

Is the correct way